src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionTooManyIndexesIntegrationTest.java

```diff
@@ -0,0 +1,110 @@
+package io.stargate.sgv2.jsonapi.api.v1;
+
+import static io.restassured.RestAssured.given;
+import static io.stargate.sgv2.common.IntegrationTestUtils.getAuthToken;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.matchesPattern;
+import static org.hamcrest.Matchers.nullValue;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.jsonapi.config.constants.HttpConstants;
+import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
+import org.junit.jupiter.api.ClassOrderer;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestClassOrder;
+
+/**
+ * Separate integration test from {@code CreateCollectionIntegrationTest} to test case of too many
+ * Indexes being created: that is, cannot create enough indexes for a new Collection.
+ */
+@QuarkusIntegrationTest
+@QuarkusTestResource(
+    value = CreateCollectionTooManyIndexesIntegrationTest.TooManyIndexesTestResource.class,
+    restrictToAnnotatedClass = true)
```

Comment:
This property is VERY important -- without it this test resources is applied to EVERY TEST CLASS!